Gemfile

```diff
@@ -1,45 +1,63 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-gem 'rake', '< 11',                      :require => false
```

Comment:
The Gemfile is generated by the PDK from: https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/Gemfile.erb

I just took the standard PDK Gemfile, if we encounter anything missing Gem, then I would add them back in, instead of having to go through all of them with a comb. But if you think it might be better to investigate every change when switching to the PDK template, I can also do that.

Reply:
Sounds good to me.